src/api/supportRequest.ts

```diff
@@ -88,7 +92,7 @@ supportRequestRoutes.post('/getNext', async (req, res) => {
     userNotified,
     supportRequest: nextRequest,
   };
-  res.send(response);
+  res.status(200).send(response);
```

Comment:
Having `.status` on there won't hurt anything, but when you use `send`, it defaults to `200`